src/renderer/components/editorWithTabs/editor.vue

```diff
@@ -3,7 +3,8 @@
     class="editor-wrapper"
     :class="[{ 'typewriter': typewriter, 'focus': focus, 'source': sourceCode }, theme]"
     :style="{ 'color': theme === 'dark' ? darkColor : lightColor, 'lineHeight': lineHeight, 'fontSize': fontSize,
-    'font-family': editorFontFamily ? `${editorFontFamily}, ${defaultFontFamily}` : `${defaultFontFamily}` }"
+    'font-family': editorFontFamily ? `${editorFontFamily}, ${defaultFontFamily}` : `${defaultFontFamily}`,
+    'direction': textDirection }"
```

Comment:
@mrg0lden @mohsenkhanpour What's the default scrollbar side? Is it on the left or right side?

Reply:
@fxha The scrollbar is on the default left side on most (almost all) of rtl pages that I have seen. Similar to ltr.
Left is also better because it doesn't gives you a unified experience when working with texts of several languages.
You don't want to look again for your scrollbar when you switch between files written in different languages.

Reply:
@fxha @enyaxu The scrollbar is on the default right side on most (almost all) of rtl pages that I have seen. Similar to ltr.
Right is also better because it gives you a unified experience when working with texts of several languages.
You don't want to look again for your scrollbar when you switch between files written in different languages.